### ISSUE TITLE
addColumn fails when order falls at end of array

### ISSUE THREAD

**Issue by patsplat** [yajra/laravel-datatables#385]
The problem appears to be in `Helper::includeInArray`.  Note how the final call in the example below returns null.

```bash
$ php -a
Interactive shell

php > require('vendor/autoload.php');
php > $item = [ 'name' => '__actions__', 'content' => 'edit-link', 'order' => 0 ];
php > $data = [ 'name' => 'username', 'id' => 1 ];
php > print_r(Yajra\Datatables\Helper::includeInArray($item, $data));
Array
(
    [__actions__] => edit-link
    [name] => username
    [id] => 1
)
php > $item = [ 'name' => '__actions__', 'content' => 'edit-link', 'order' => 1 ];
php > $data = [ 'name' => 'username', 'id' => 1 ];
php > print_r(Yajra\Datatables\Helper::includeInArray($item, $data));
Array
(
    [name] => username
    [__actions__] => edit-link
    [id] => 1
)
php > $item = [ 'name' => '__actions__', 'content' => 'edit-link', 'order' => 2 ];
php > $data = [ 'name' => 'username', 'id' => 1 ];
php > print_r(Yajra\Datatables\Helper::includeInArray($item, $data));
php > 
```

Installed `"yajra/laravel-datatables-oracle": "~6.0"`

**Comment by patsplat** [comment]
Submitted pull request with test case + fix.  Please review at your earliest convenience.

Thanks!

**Comment by yajra** [comment]
Whoops! Can you please re-submit the PR and target 6.0 branch? develop branch is not updated. Thanks!

**Comment by yajra** [comment]
Released on v6.4.2. Thanks a lot!

**Comment by patsplat** [comment]
np thanks!
